.travis/travis-openmandriva.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pushd $SCRIPT_DIR
+
+source $SCRIPT_DIR/travis-common.inc
+
+set -e
+set -x
+
+JOB_NAME=${TRAVIS_JOB_NAME:-OpenMandriva Lx cooker }
```

Comment:
```suggestion
JOB_NAME=${TRAVIS_JOB_NAME:-OpenMandriva cooker }
```

I'm not sure if that was a typo, but the code value below expects the second word to be the release value.

Reply:
Fixed